src/adm/tiles.rs

```diff
@@ -245,6 +245,12 @@ pub async fn get_tiles(
                 })?
         }
     };
+    metrics.distribution_with_tags(
+        "tiles.adm.response.distribution",
```

Comment:
I don't care if 'distribution' is included in this name but maybe something more specific:
```suggestion
        "tiles.adm.response.tiles_count",
```